docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -42,6 +42,8 @@ key that the user is not allowed to access will display an _Access Denied_ messa
 * Prefix-aware search performs a head request on every object result; this may slow
 search performance in the Catalog
 
+* Number of search results is not available with prefix-aware search
```

Comment:
```suggestion
* The Catalog does not (accurately) display the number of search results
```

Reply:
this limitation applies only to prefix-aware search aka secure search and you suggestion remove its mention